src/shrutiC/Shop.java

```diff
@@ -0,0 +1,99 @@
+package shrutiC;
+
+class Shop {
+
+	static int maggieStock = 50;
+	static int dosaStock = 43;
+	static int oilPouchStock = 39;
+	static int panipuriStock = 43;
+	static int masalaStock = 73;
+
+	void displayOutOfStockItem() {
+
+		System.out.println("\nSorry below items are Out of stock :-");
+		if (maggieStock == 0)
+			System.out.println("Maggie");
+		if (panipuriStock == 0)
+			System.out.println("Panipuri");
+		if (oilPouchStock == 0)
+			System.out.println("Oil pouch");
+		if (dosaStock == 0)
+			System.out.println("Dosa");
+		if (masalaStock == 0)
+			System.out.println("Masala");
+
+	}
+
+	void displayAvailableItem() {
+
+		System.out.println("Below items are available in stock :-");
+		if (maggieStock > 0)
+			System.out.println("Maggie - " + maggieStock + " packets");
+		if (panipuriStock > 0)
+			System.out.println("Panipuri - " + panipuriStock + " packets");
+		if (oilPouchStock > 0)
+			System.out.println("Oil Pouch - " + oilPouchStock + " packets");
+		if (dosaStock > 0)
+			System.out.println("Dosa - " + dosaStock + " packets");
+		if (masalaStock > 0)
+			System.out.println("Masala - " + masalaStock + " packets");
+
+	}
+
+	void orderMaggie(int maggieOrder) {
+
+		if (maggieOrder <= maggieStock) {
+			maggieStock = maggieStock - maggieOrder;
+		} else if (maggieOrder > maggieStock && !(maggieStock == 0)) {
+			maggieStock = 0;
```

Comment:
Assume we have 51 maggieOrder then it will go into else if condition, but condition you give bit wrong logically.
(51>50) && !(51==0)
true       &&     true

if condition satisfied then you are making maggieStock  = 0 which wrong.

same for other methods.